spec/services/forest_liana/resources_getter_spec.rb

```diff
@@ -54,11 +68,80 @@ def init_scopes
       init_scopes
```

Comment:
```suggestion
      manufacturers = [
        { :name => 'Orange' },
        { :name => 'Pear'}
      ].map { |manufacturer| Manufacturer.create(name: manufacturer[:name])}

      drivers = [
        { :firstname => 'Baby driver' },
        { :firstname => 'Taxi driver'}
      ].map { |driver| Driver.create(firstname: driver[:firstname])}

      products = [
        { :name => 'Valencia', :uri => 'https://valencia.com', :manufacturer => manufacturers[0], :driver => drivers[0] },
        { :name => 'Blood', :uri => 'https://blood.com', :manufacturer => manufacturers[0], :driver => drivers[1] },
        { :name => 'Conference', :uri => 'https://conference.com', :manufacturer => manufacturers[1], :driver => drivers[0] },
        { :name => 'Concorde', :uri => 'https://concorde.com', :manufacturer => manufacturers[1], :driver => drivers[1] },
      ].map { |product| Product.create(name: product[:name], uri: product[:uri], manufacturer: product[:manufacturer], driver: product[:driver])}
      
      init_scopes
```

Reply:
Using index references to records in an array seems a bit more complicated to parse than using named records (I know exactly what the `orange_manufacturer` is, it's hard for me to figure out what `manufacturers[2]`), but if you prefer that format I'll stick to it then.

Reply:
You are totally right, it was just a matter of regrouping records creation for a test suit. I have adapted the code to correspond to your remark 👌